Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Vuejs Frontend
+FROM node:lts-alpine as build-stage
+WORKDIR /front
+COPY front/ .
+RUN npm install
+RUN npm run build
+
+# Python Backend
+FROM python:3.10-slim-buster
```

Comment:
The 3.10 python version doesn't match our requirements

Reply:
Well seen, thanks, fixed in [de9d68c](https://github.com/Museum-GO/MuseumGO-App/pull/32/commits/de9d68c1c9d39ccd28b57890386f3f9b86344d25)